### PR TITLE
295396 update install adapter section

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/main/shared-content/installation/install-using-docker.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+uid: InstallUsingDocker
+---
+
+# Installation using Docker
+
+Docker is a set of tools that you can use on Linux to manage application deployments. This topic provides examples of how to create a Docker container with the adapter.
+
+**Notes:** 
+
+- If you want to use Docker, you must be familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter.
```
Comment (KNyce71) [inline content/main/shared-content/installation/install-using-docker.md:11]:
```suggestion
- If you want to use Docker, OSIsoft recommends that you be both familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter. 
```

**Review by KNyce71 — COMMENTED** [review]
One minor suggestion identical to the one in PR #36

File: content/toc.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@
   items:
   - topicUid: SystemRequirements
   - topicUid: InstallTheAdapter
-  - topicUid: InstallPIAdapterForMQTTUsingDocker
+  - topicUid: InstallUsingDocker
   - topicUid: UninstallTheAdapter
```
Comment (rmuolic) [inline content/toc.yml:12]:
```suggestion
  - topicUid: InstallUsingDocker
  - topicUid: UpgradeTheAdapter
  - topicUid: UninstallTheAdapter
```

File: content/main/shared-content/_includes/inline/product-name.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+PI Adapter for MQTT
+Search Documentation
+
```
Comment (rmuolic) [inline content/main/shared-content/_includes/inline/product-name.md:3]:
```suggestion
```

**Review by rmuolic — CHANGES_REQUESTED** [review]
Minor changes req.

File: content/main/shared-content/installation/install-using-docker.md (hunk 1 above)
Comment (rmuolic) [inline content/main/shared-content/installation/install-using-docker.md]:
```suggestion
- The use of Docker is only recommended if your environment requires it. Only users proficient with Docker should use it to install the adapter. Docker is not required to use the adapter.
```
```suggestion
- If you want to use Docker, you must be familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter. 
```

Comment (rmuolic) [inline content/main/shared-content/installation/install-using-docker.md]:
```suggestion
- The use of Docker is only recommended if your environment requires it. Only users proficient with Docker should use it to install the adapter. Docker is not required to use the adapter.
```

**Review by rmuolic — CHANGES_REQUESTED** [review]
Docker verbiage update.

File: content/main/shared-content/installation/install-using-docker.md (hunk 1 above)
Comment (osisoft-mbishop) [reply content/main/shared-content/installation/install-using-docker.md]: Updated. I couldn't accept your suggestion for some reason though, so I did it manually.

**Review by rmuolic — APPROVED** [review]

**Pull request merged**